docs/src/catalyst_applications/homotopy_continuation.md

```diff
@@ -8,136 +8,75 @@ example those which are purely mass action or have only have Hill functions with
+Here, we only run `import HomotopyContinuation` as we do not require any of its functions, and just need it to be present in the current scope for the extension to be activated.
 
-subs = Dict(MT.parameters(ns) .=> pvals)
-neweqs = map(eq -> substitute(eq.rhs, subs), equations(ns))
-```
-Finally, we use Catalyst's `to_multivariate_poly` function to reinterpret our
-symbolic equations in a polynomial representation that is compatible with
-HomotopyContinuation. We can then apply HomotopyContinuation's `solve` command
-to find the roots, using `real_solutions` to filter our non-physical complex
-steady-states:
+Now we can find the steady states using:
 ```@example hc1
-polyeqs = Catalyst.to_multivariate_poly(neweqs)
-sols = HC.real_solutions(HC.solve(polyeqs))
+hc_steady_states(wilhelm_2009_model, ps)
 ```
-Note that HomotopyContinuation orders variables lexicographically, so this will
-be the ordering present in each steady-state solution vector (i.e. `[X1, X2]` is
-the ordering here).
+The order of the species in the output vectors are the same as in `species(wilhelm_2009_model)`.
+
+It should be noted that the steady state multivariate polynomials produced by reaction systems may have both imaginary and negative roots, which are filtered away by `hc_steady_states`. If you want the negative roots, you can use the `hc_steady_states(wilhelm_2009_model, ps; filter_negative=false)` argument.
 
-While it is not the case for this CRN, we note that solutions with negative
-species concentrations can be valid (unphysical) steady-states for certain
-systems. These will need to be filtered out as well.
 
 ## Systems with conservation laws
-Finally, some systems are under-determined, and have an infinite number of
-possible steady states. These are typically systems containing a conservation
+Some systems are under-determined, and have an infinite number of possible steady states. These are typically systems containing a conservation
 law, e.g.
 ```@example hc3
 using Catalyst
 import HomotopyContinuation
-const MT = ModelingToolkit
-const HC = HomotopyContinuation
 
 two_state_model = @reaction_network begin
     (k1,k2), X1 <--> X2
 end
 ```
-Catalyst allows the conservation laws to be computed using the
-`conservationlaws` function. By using these to reduce the dimensionality of the
-system, as well specifying the initial amount of each species,
-HomotopyContinuation can again be used to find steady states. First, we set the
-default values of the system's initial conditions and parameter values. This
-will allow the system to automatically find the conserved amounts.
-```@example hc3
-setdefaults!(two_state_model, [:X1 => 1.0, :X2 => 1.0, :k1 => 2.0, :k2 => 1.0])
-```
-Next, we create a `NonlinearSystem`, while also removing one species via the
-conservation equation.
+To find the steady states of these, an initial condition must also be provided (which is used to compute the system's conserved quantities, in this case `X1+X2`):

```

Comment:
I think we should still say what Catalyst is doing under the hood before this. You can say it is done automatically, but we should advertise this functionality is in Catalyst...

Reply:
We want people to appreciate that this isn't something done by HC for them, but is a Catalyst feature / functionality that many packages are lacking.